apps/src/code-studio/pd/form_components/button_list.jsx

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import _ from 'lodash';
-import {Radio, Checkbox, ControlLabel, FormGroup, HelpBlock} from 'react-bootstrap';
+import {
```

Comment:
[it is](https://github.com/airbnb/javascript#modules--multiline-imports-over-newlines), but I unfortunately don't think eslint has an option to complain about this